src/test/integration/controllers/accounts.controller.e2e-spec.ts

```diff
@@ -0,0 +1,1864 @@
+                .get(`${path}/${address}/verification`)
+                .expect(200)
+                .then(res => {
+                    expect(res.body.codeHash).toBeDefined();
+                    expect(res.body.source).toBeDefined();
+                    expect(res.body.status).toStrictEqual("success");
+                    expect(res.body.ipfsFileHash).toBeDefined();
+                });
+        });
+
+        it('should throw exception "Account verification not found"', async () => {
+            const address: string = 'erd1lz6vlamhv849yfhp4dyk9362rga3zx24rt45vczuc4q5wsgk5vksxmq7yv';
+
+            await request(app.getHttpServer())
+                .get(`${path}/${address}/verification`)
+                .expect(404)
+                .then(res => {
+                    expect(res.body.message).toStrictEqual("Account verification not found");
+                });
+        });
+
+        it('should return 400 Bad Request for an invalid address', async () => {
+            const address: string = 'erd1qqqqqqqk8dal6wje72csqma5j6ncguwj4zvdqc6xzwj';
+
+            await request(app.getHttpServer())
+                .get(`${path}/${address}/verification`)
+                .expect(400)
+                .then(res => {
+                    expect(res.body.message).toStrictEqual("Validation failed for argument 'address' (a bech32 address is expected)");
+                });
+        });
+    });
+
+    describe('/accounts/{address}/tokens', () => {
+        it('should return 400 Bad Request for an invalid address', async () => {
+            const address: string = 'erd1qqqqqqqk8dal6wje72csqma5j6ncguwj4zvdqc6xzwj';
+
+            await request(app.getHttpServer())
+                .get(`${path}/${address}/tokens`)
+                .expect(400)
+                .then(res => {
+                    expect(res.body.message).toStrictEqual("Validation failed for argument 'address' (a bech32 address is expected)");
+                });
+        });
+
+        it(`should return a list of all available fungible tokens for a given address, together with their balance`, async () => {
+            const address: string = 'erd1rf4hv70arudgzus0ymnnsnc4pml0jkywg2xjvzslg0mz4nn2tg7q7k0t6p';
+
+            await request(app.getHttpServer())
+                .get(`${path}/${address}/tokens`)
+                .expect(200)
+                .then(res => {
+                    expect(res.body).toBeInstanceOf(Array<TokenWithBalance>);
+                });
+        });
+
+        [
+            {
+                filter: 'includeMetaESDT',
+                value: 'true',
+            },
+            {
+                filter: 'includeMetaESDT',
+                value: 'false',
+            },
+        ].forEach(({ filter, value }) => {
+            describe(`when filter ${filter} is applied`, () => {
+                if (value === 'true') {
+                    it(`should returns a list of all available fungible and meta tokens for a given address`, async () => {
```

Comment:
miss typo `returns` -> return

Reply:
Also, replace test description with:
should return a list of all available Fungible / MetaESDT tokens for a given address